portfolio/src/main/webapp/about.js

```diff
@@ -1,82 +1,106 @@
-function loadInFact(index, fact_index, isTruth) {
-    let fact = isTruth ? truths[fact_index] : lies[fact_index];
+let correctAnswer = -1;
 
-    switch(index) {
-        case 0:
-            document.getElementById("first_txt").innerHTML = fact[0];
-            document.getElementById("first_img").src = fact[1];
-            break;
-        case 1:
-            document.getElementById("second_txt").innerHTML = fact[0];
-            document.getElementById("second_img").src = fact[1];
-            break;
-        case 2:
-            document.getElementById("third_txt").innerHTML = fact[0];
-            document.getElementById("third_img").src = fact[1];
-            break;
-    }
+/**
+ * Advances the game to the next question. Loads in the first question 
+ * game has not yet started. Otherwise, checks answer against user 
+ * selected anwer and alerts if a lie or truth has been chosen.
+ */
+function next()  {
+    let gameStatus = document.getElementById('game-status');
+    if (gameStatus.innerHTML=='Press Start to Begin!') {
```

Comment:
First, add spacing on the outsides of the equal signs here.

Also, try to always use the triple equals (===) instead of the double equals in javascript.  Haha... this is a "fun" complexity of js if you haven't run into it before.  But basically, triple equals is more specific, whereas double equals can be a little... "meh, close enough for me".

For some examples, the following will evaluate to true:
0 == ''
'' == false
'0' == 0
null == undefined

Using triple equals will ensure that your variables are the correct type as well so that your code will never accidentally think that say, an empty string is equal to the number zero.

https://google.github.io/styleguide/jsguide.html#features-equality-checks

Reply:
Interesting!